### PR TITLE
Fix loading IdentityUserByUserName for migration to v6

### PR DESCRIPTION
**Pull request by abremora**
#11: Fix loading IdentityUserByUserName in UserStore.MigrateToV6() because of hard coded document prefix.

### CODE REVIEW

File: RavenDB.Identity/UserStore.cs [hunk 1]
```diff
@@ -798,12 +798,14 @@ public Task<int> CountCodesAsync(TUser user, CancellationToken cancellationToken
...
         {
+            var collectionName = docStore.Conventions.FindCollectionName(typeof(IdentityUserByUserName));
+
             // Step 1: find all the old IdentityByUserName objects.
             var emails = new List<(string userId, string email)>(1000);
             using (var dbSession = docStore.OpenSession())
             {
 #pragma warning disable CS0618 // Type or member is obsolete
-                var stream = dbSession.Advanced.Stream<IdentityUserByUserName>("IdentityUserByUserNames/");
+                var stream = dbSession.Advanced.Stream<IdentityUserByUserName>("{collectionName}/");
```
Comment (JudahGabriel) [inline RavenDB.Identity/UserStore.cs]: Did you mean $"{collectionName}/" (notice the dollar sign)?

File: RavenDB.Identity/UserStore.cs [hunk 2]
```diff
@@ -832,7 +834,7 @@ public static void MigrateToV6(IDocumentStore docStore)
                 .Operations
                 .Send(new DeleteByQueryOperation(new Client.Documents.Queries.IndexQuery
                 {
-                    Query = "from IdentityUserByUserNames"
+                    Query = "from {collectionName}"
```
Comment (JudahGabriel) [inline RavenDB.Identity/UserStore.cs]: Did you mean to do string interpolation here?

File: RavenDB.Identity/UserStore.cs (hunk 1 above)
Comment (abremora) [reply RavenDB.Identity/UserStore.cs]: You are right. Fixed.

File: RavenDB.Identity/UserStore.cs (hunk 2 above)
Comment (abremora) [reply RavenDB.Identity/UserStore.cs]: You are right. Fixed.

**Pull request merged**